03 Writing Algorithms/03 Securities/99 Asset Classes/06 Forex/04 Market Hours/11 OANDA/09 Supported Assets.html

```diff
@@ -0,0 +1,18 @@
+<!-- Code generated by market-hour-convertor.py -->
+<p>The following table shows the available contracts in the OANDA Forex market. Some of these contracts may have different trading periods than the overall OANDA Forex market.</p>
+
+<table class="table qc-table table-reflow">
+<thead>
+<tr><th>Symbol</th><th>Contract</th></tr>
+</thead>
+<tbody>
+<tr><td><a href="oanda/eurnzd">EURNZD</a></td><td>EUR/NZD</td></tr>
+<tr><td><a href="oanda/gbpnzd">GBPNZD</a></td><td>GBP/NZD</td></tr>
+<tr><td><a href="oanda/nzdcad">NZDCAD</a></td><td>NZD/CAD</td></tr>
+<tr><td><a href="oanda/nzdchf">NZDCHF</a></td><td>NZD/CHF</td></tr>
+<tr><td><a href="oanda/nzdhkd">NZDHKD</a></td><td>NZD/HKD</td></tr>
+<tr><td><a href="oanda/nzdjpy">NZDJPY</a></td><td>NZD/JPY</td></tr>
+<tr><td><a href="oanda/nzdsgd">NZDSGD</a></td><td>NZD/SGD</td></tr>
+<tr><td><a href="oanda/nzdusd">NZDUSD</a></td><td>NZD/USD</td></tr>
```

Comment:
Is this table missing some pairs? [Supported Assets](https://www.quantconnect.com/docs/v2/writing-algorithms/datasets/oanda/forex-data#05-Supported-Assets) shows more pairs.

If the remaining pairs use the "generic" market hours, we should add a sentence underneath the table that explains it.

Reply:
The problem is the text above: "_The following table shows the available contracts in the OANDA Forex market. Some of these contracts may have different trading periods than the overall OANDA Forex market._" which is not correct. It should be something like: "_The following pairs have different trading periods than the overall Forex market._"

We shouldn't use "/" to delimit the currencies since they are never referred to like this in Lean. Also, showing that "GBPNZD" represents "GBP/NZD" doesn't add useful information.

Reply:
`GBP/NZD` this slashed representation is the actual instrument name in OANDA (see https://github.com/QuantConnect/Lean/blob/master/Data/symbol-properties/symbol-properties-database.csv and https://www.oanda.com/sg-en/trading/cfds/forex/)